.gitlab-ci.yml

```
@@ -75,7 +75,7 @@ stages:
     include:
       - local: '.gitlab/custom-jobs-and-variables.yml'
       - project: 'radiuss/radiuss-shared-ci'
-        ref: 'v2024.04.0'
+        ref: 'v2024.06.0'
```

Are we going with 07 or 06?

```suggestion
        ref: 'v2024.07.0'
```

---

That's the tag for the radiuss-shared-ci repo. It's independent of the RAJA release version.